common/VotingUtils.js

```diff
@@ -177,7 +177,24 @@ const batchRetrieveRewards = async (requests, roundId, votingContract, account)
   };
 };
 
+// Get the latest event matching the provided parameters. Assumes that all events from Voting.sol have indexed
+// parameters for identifier, roundId, and voter.
+const getLatestEvent = async (eventName, request, roundId, account, votingContract) => {
+  const events = await votingContract.getPastEvents(eventName, {
+    fromBlock: 0,
+    filter: { identifier: request.identifier, roundId: roundId.toString(), voter: account.toString() }
```

Comment:
I really like this function!

For another PR, but we probably should change the all of the events in `Voting.sol` to refer to have the same names for these parameters. `PriceRequestAdded` and `PriceResolved` have different names for `roundId`. Even though those two methods *don't* have a `voter` argument, this function should still work because that filter parameter would just be ignored by `web3`. I mention this because I think this method might be a super convenient and universal way to interface with the Voting contract's events.